pkgs/development/web/bun/update-script

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq gawk
+# vim:ft=bash
+
+set -eu -o pipefail
+
+LATEST_VERSION=$(curl https://api.github.com/repos/oven-sh/bun/releases/latest | jq .tag_name -r | sed 's;bun-v\([0-9]*\.[0-9]*\.[0-9]*\);\1;g')
+
+# nix-instantiate --eval --json -E $(
+NIX_DIST=$(cat default.nix | awk '
```

Comment:
A useless cat boldly going where no nix package has gone before!

Instead of `cat file | awk script` you should simply do `awk -e <expression> file`